### PR TITLE
upgraded golang image version to latest.

### PR DESCRIPTION
**Pull request by yanggangtony**
Signed-off-by: yanggang <gang.yang@daocloud.io>

**Please provide a description of this PR:**
Upgraded golang version in the dockerfile to build.

/label release-notes-none

```release-note
None
```

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]

**Review by howardjohn — APPROVED** [review]

File: samples/jwt-server/src/Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@
 #   limitations under the License.
 
 # build a jwt-server binary using the golang container
-FROM golang:1.19 as builder
+FROM golang:1.19.4 as builder
```
Comment (ericvn) [inline samples/jwt-server/src/Dockerfile:16]: This change is not needed. The 1.19 tag is shared with the latest 1.19.x tag, currently 1.19.4,

File: samples/tcp-echo/src/Dockerfile [hunk 2]
```diff
@@ -16,7 +16,7 @@
 ARG BASE_DISTRIBUTION=default
 
 # build a tcp-echo binary using the golang container
-FROM golang:1.14.2 as builder
+FROM golang:1.19.4 as builder
```
Comment (ericvn) [inline samples/tcp-echo/src/Dockerfile:19]: Better to make this 1.19.

**Review by ericvn — CHANGES_REQUESTED** [review]

File: samples/tcp-echo/src/Dockerfile (hunk 2 above)
Comment (yanggangtony) [reply samples/tcp-echo/src/Dockerfile]: thanks for review.
Done it.

File: samples/jwt-server/src/Dockerfile (hunk 1 above)
Comment (yanggangtony) [reply samples/jwt-server/src/Dockerfile]: thanks for review.
Done it.

**Comment by yanggangtony** [comment]
Hey, I want to know how to add a lable for the release-note-none?
Is that like k8s 's style?

```release-note
None
```

**Review by ericvn — APPROVED** [review]

**Pull request merged**